omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@guzzilar At first, it has only one class, `OmisePaymentModuleFrontController`. [This class does all functional to create the charge](https://github.com/omise/omise-prestashop/blob/bc8b1d323ef63866574590989ee278a5d5170448/omise/controllers/front/payment.php). It has many dependencies and functionalities.

Along with the rise of the features that will be developed soon, the class `OmisePaymentModuleFrontController` will has more size, logic and complexity that adversely affect to the maintenance. For an example from another project, [function process_payment()](https://github.com/omise/omise-woocommerce/blob/master/omise-wc-gateway.php#L88).

So, a class `OmisePaymentModuleFrontController ` has been reduced the dependencies to be one dependency (`Charge`).

Including, PrestaShop has provided the internal functions to retrieve the information such as cart, currency and etc. These functions enable class `Charge` to retrieve all information that need to create Omise charge by itself. Which it does not need to pass from class `OmisePaymentModuleFrontController`.

The knowledge of how to charge has been packed in `Charge` class.
The controller class `OmisePaymentModuleFrontController` is control the flow.

How about a class `Charge` has been renamed to `OmiseChargeModuleFrontController`?